src/main/java/seedu/address/storage/JsonAdaptedModuleClass.java

```diff
@@ -20,20 +22,20 @@
  */
 public class JsonAdaptedModuleClass {
 
-    public static final String MISSING_FIELD_MESSAGE_FORMAT = "ModuleClass's %s field is missing!";
-    public static final String INVALID_FIELD_MESSAGE_FORMAT = "ModuleClass's %s field is invalid!";
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Class's %s field is missing!";
+    public static final String MESSAGE_INVALID_STUDENTS_IN_LESSON = "Invalid student(s) found in lesson(s).";
+    public static final String INVALID_FIELD_MESSAGE_FORMAT = "Class's %s field is invalid!";
```

Comment:
I think should be `Class'`.
```suggestion
    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Class' %s field is missing!";
    public static final String MESSAGE_INVALID_STUDENTS_IN_LESSON = "Invalid student(s) found in lesson(s).";
    public static final String INVALID_FIELD_MESSAGE_FORMAT = "Class' %s field is invalid!";
```